google-cloud-spanner/src/main/java/com/google/cloud/spanner/v1/stub/metrics/MetricRegistryConstants.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spanner.v1.stub.metrics;
+
+import com.google.common.collect.ImmutableList;
+import io.opencensus.metrics.LabelKey;
+import io.opencensus.metrics.LabelValue;
+
+/** A helper class that holds OpenCensus's related constants. */
+public class MetricRegistryConstants {
+
+  // The label keys are used to uniquely identify timeseries.
+  private static final LabelKey DATABASE = LabelKey.create("database", "Target database");
+  private static final LabelKey INSTANCE_ID =
+      LabelKey.create("instance_id", "Name of the instance");
+  private static final LabelKey LIBRARY_VERSION =
+      LabelKey.create("library_version", "Library version");
+
+  /** The label value is used to represent missing value. */
+  private static final LabelValue UNSET_LABEL = LabelValue.create(null);
+
+  public static final ImmutableList<LabelKey> SPANNER_LABEL_KEYS =
+      ImmutableList.of(DATABASE, INSTANCE_ID, LIBRARY_VERSION);
+
+  public static final ImmutableList<LabelValue> SPANNER_DEFAULT_LABEL_VALUES =
+      ImmutableList.of(UNSET_LABEL, UNSET_LABEL, UNSET_LABEL);
+
+  /** Unit to represent counts. */
+  public static final String COUNT = "1";
+
+  // The Metric name and description
+  public static final String ACTIVE_SESSIONS = "cloud.google.com/java/spanner/active_sessions";
+  public static final String MAX_SESSIONS = "cloud.google.com/java/spanner/max_sessions";
+  public static final String ACTIVE_SESSIONS_DESCRIPTION = "The number of active sessions";
```

Comment:
I think the description of this metric should be different, considering it is reading `maxSessionsInUse`. This value will only increase during a 10 minute interval, and then be reset to zero (and then start increasing again). Suggestion, something like:
'Max number of sessions in use during the last 10 minutes'.

I noticed the other discussion about whether to measure `numSessionsInUse` or `maxSessionsInUse`. I think both could be valuable for the user, and `numSessionsInUse` should have a description like 'The number of sessions currently in use'.

Reply:
Sure, will change the description for `maxSessionsInUse`. Could you please elaborate more on `numSessionsInUse` metric. How is it different from  `maxSessionsInUse` and how do you think it is useful for users in conjunction with `maxSessionsInUse`? 

Just so you know, we don't want to add too many metrics due to costs associated with exporting to backend (like stackdriver). Having said that, I am more than happy to include metrics which are meaningful (and useful) for the users to resolve/understand the production issues at some extend.

Reply:
* `numSessionsInUse` reflects the number of sessions checked out from the pool at this very moment. Depending on probe intervals and average transaction duration, this number might often be measured to be zero for a system under low load. E.g. if the system executes 1 transaction per second, the average transaction duration is 300ms and the value is probed once every 10 seconds, the chances are big that you will often measure the value between two transactions and get a zero. On a system with higher load, measuring this value will give the user a better idea about how the distribution of the transactions are over time. If this value is relatively constant most of the time, but suddenly shows a spike that cannot be correlated with a similar sudden spike in user requests, it might be an indication of a concurrency problem somewhere (locks in the database, concurrency issues in the application itself etc.).
* `maxSessionsInUse` reflects the maximum number of `numSessionsInUse` during the current maintenance window. A maintenance window is a hard-coded 10 minute interval. After a complete maintenance window has passed, the value is reset to 0. The value is updated every time a session is checked out of the pool by the simple calculation `maxSessionsInUse = Math.max(numSessionsInUse, maxSessionsInUse)`.

As you can conclude from the above (and which I also just now realized ;-)), the `maxSessionsInUse` has a slight flaw: If no new session is checked out during a maintenance window, the value will remain zero during the entire window, even if there are sessions that are kept checked out during the entire window (i.e. the sessions were checked out during a previous maintenance window).

Reply:
Thanks for the explanation. I have added `numSessionsInUse` metric PTAL.

Reply:
I would like to get this PR merged before adding other metrics.